packages/jest-mock/src/__tests__/symbolsAreCool-test.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
```

Comment:
can you call this file `symbols-test.js`? I don't like having funny names in here. Also, this file (and the other one) will need a copyright header.